internal/duration/duration.go

```diff
@@ -0,0 +1,68 @@
+// Package duration defines the various durations used throughout
+// Carrier, as timeouts, and other.
+package duration
+
+import (
+	"time"
+)
+
+const (
+	multiplier    = 1
```

Comment:
I don't see a way to change the multiplier are run time. Maybe we can create a `Multiplier()` method that return either the value of `CARRIER_TIMEOUT_MULTIPLIER` environment variable or `1` if the variable is not set. What do you think?

Reply:
Heh. I had defered that yesterday evening to today. The commit is currently waiting to be made (just a bit of testing).
The variable in the commit is `CARRIER_DURATION_MULTIPLIER` ... Hm. Timeout fits better, all the affected durations are timeouts (poll interval and user abort are not affected, and not timeouts per se). ... Will change.